src/main/java/Task/Deadline.java

```diff
@@ -0,0 +1,23 @@
+package Task;
+
+public class Deadline extends Task {
+    private String by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        setBy(by);
+    }
+
+    public void setBy(String by) {
+        this.by = by;
+    }
+
+    public String getBy() {
+        return by;
+    }
+
+    @Override
+    public String toString() {
+        return "[D]" + super.toString() + "(by: " + getBy() + ")";
```

Comment:
Should "D" be defined as a final string variable? I see this in other places as well.